languages/csharp/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,45 @@
+In C#, data structures that can hold zero or more elements are known as _collections_. An **array** is a collection that has a fixed size and whose elements must all be of the same type. Elements can be assigned to an array or retrieved from it using an index. C# arrays are zero-based, meaning that the first element's index is always zero:
+
+```csharp
+// Declare array with explicit size (size is 2)
+int[] twoInts = new int[2];
+
+// Assign first and second element by index
+twoInts[0] = 7;
+twoInts[1] = 8;
+
+// Retrieve the second element by index
+twoInts[1] == 8; // => true
+
+// Check the length of the array
+twoInts.Length == 2; // => true
+```
+
+Arrays can also be defined using a shortcut notation that allows you to both create the array and set its value. As the compiler can now tell how many elements the array will have, the length can be omitted:
+
+```csharp
+// Assign and initialize (size is 3)
+int[] threeIntsV1 = new int[] { 4, 9, 7 };
+
+// Shorter notation (size is 3)
+int[] threeIntsV2 = new[] { 4, 9, 7 };
+
+// Shortest notation (size is 3)
+int[] threeIntsV3 = { 4, 9, 7 };
+```
+
+Arrays can be manipulated by either calling an array's methods or properties, or by using the methods defined in the `Array` class.
```

Comment:
```suggestion
Arrays can be manipulated by using the methods and properties defined in the `Array` class.
```

Reply:
This suggestion is not true. There _are_ properties and instance methods on an array, such as the `Length` property and the `CopyTo()` method, and it is important to know that there are.

The fact that an array has methods becomes especially important later on, when students will know about LINQ's extension methods, which also apply to arrays.

Reply:
And these methods and  properties are defined on the array class right? 
Or you imply on the second half of the sentence that there are static methods on the Array class?

I just think that the sentence is confusing, and it links three times to the array class...  and it looks like the message is just that an array is an object of the Array class, then everything else is implied.

Reply:
Ah right, I didn't get what you were saying. I've updated the documentation to make the distinction more clear. Sorry for the confusion.